bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -1537,7 +1537,22 @@
     */
    @Transient
    public boolean             isBackupImportFile;
-   
+
+   // ############################################# CUSTOM TRACKS TRANSIENT#######################################
+   @Transient
+   HashMap<String, float[]> customTracks = new HashMap<>();
+
+   @Transient
+   public HashMap<String, CustomTrackDefinition> customTracksDefinition = new HashMap<>();
+
+   @Transient
+   HashMap<String, float[]> _customTracks_UI = new HashMap<>();
```

Comment:
ok I'll change that in my next PR which will contain the timeslice view chart view for Custom Tracks

Reply:
changing the name of th class "net.tourbook.data.CustomTrackStatEntry" to  "net.tourbook.data.CustomTrackStatisticEntry" will cause a class excpetion when deserializin asaved data under "net.tourbook.data.CustomTrackStatEntry". but I assume no one has already stored custom tracks; so it should be ok right ?

Reply:
> @wolfgang-ch changing the name of the class "net.tourbook.data.CustomTrackStatEntry" to "net.tourbook.data.CustomTrackStatisticEntry" will cause a class exception when deserializing already saved data under "net.tourbook.data.CustomTrackStatEntry". but I assume no one has already stored custom tracks in DB; so it should be ok right ?

Your code is not yet merged into the main branch, so there are no dependencies and you can modify what you want

Reply:
ah ok, I thought you said  that we can already see "custom tracks" in the "tour info view" here: https://github.com/wolfgang-ch/mytourbook/pull/280#issuecomment-756127146

Reply:
> ah ok, I thought you said that we can already see "custom tracks" in the "tour info view" here: [#280 (comment)](https://github.com/wolfgang-ch/mytourbook/pull/280#issuecomment-756127146)

When I tested your branch in my IDE then I saw the custom tracks in the tour data view. 

Until a new version is published (when I send the translators a test version which is about 2 weeks before it is published) then everything can be changed, but after that, the data structure is frozen otherwise there could be a lot of trouble